recipes/libpng/all/conanfile.py

```diff
@@ -108,6 +107,16 @@ def _neon_msa_sse_vsx_mapping(self):
             "check": "check",
         }
 
+    @property
+    def _libpng_cmake_system_processor(self):
+        # FIXME: too specific and error prone, should be delegated to a conan helper function
+        # It should satisfy libpng CMakeLists specifically, do not use it naively in an other recipe
+        if "mips" in self.settings.arch:
+            return "mipsel"
+        if "ppc" in self.settings.arch:
+            return "powerpc"
+        return str(self.settings.arch)
```

Comment:
For review: upstream CMakeLists doesn't require to be too specific about CMAKE_SYSTEM_PROCESSOR.
For example armv8 doesn't need to be mapped to aarch64, armv8 will just work because they check `^arm` or `^aarch64`. 32 bits and 64 bits architectures have the same logic in their CMakeLists.